app/controllers/guests_controller.rb

```diff
@@ -0,0 +1,8 @@
+class GuestsController < ApplicationController
+  def show
+    @guest = Guest.find(params[:id])
+    @rooms = @guest.rooms
```

Comment:
Keep in mind that you don't necessarily need to set up the @rooms instance var here as you can reference it in the view as @guest.rooms